src/attributes.jl

```diff
@@ -23,6 +23,23 @@ abstract type AbstractConstraintAttribute end
 
 const AnyAttribute = Union{AbstractInstanceAttribute, AbstractVariableAttribute, AbstractConstraintAttribute}
 
+"""
+    supportsattribute(instance::AbstractInstance, attr::AbstractInstanceAttribute, ::Type{V}) where V
+
+Return a `Bool` indicating whether there exists an instance `src` that contains the attribute `attr` and `get(src, attr)` is of type `V` such that `copy!(instance, src)` returns a `CopyResult` object with a status `CopySuccess`.
```

Comment:
"There exists" is confusing. What does this mean?

Reply:
This is an attempt of rigorous version of "`instance` support the attribute/constraint except eventually on some special case such as constraints that cannot be together in the same instance`. With the "There exists" we only require that there is at least one situation in which the constraint is supported. Therefore, if this return false, it means that the presence if such constraint/attribute in `src` will make it impossible to copy it into `instance` whatever is the rest of `src`.